CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.0)
 project(cityflow)
 
 set(CMAKE_CXX_STANDARD "11" CACHE STRING "")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -DRAPIDJSON_HAS_STDSTRING=1")
```

Comment:
This argument is only used when building cityflow_lib, right? If so, can you make this argument to the specific target, rather than being a global one?